hack/ginkgo-e2e.sh

```diff
@@ -89,7 +89,7 @@ fi
 export PATH=$(dirname "${e2e_test}"):"${PATH}"
 "${ginkgo}" "${ginkgo_args[@]:+${ginkgo_args[@]}}" "${e2e_test}" -- \
   "${auth_config[@]:+${auth_config[@]}}" \
-  --host="https://${KUBE_MASTER_IP-}" \
+  --host="https://${KUBE_MASTER_IP:-}" \
```

Comment:
Assuming you want this to be `--host="https://"` when `$KUBE_MASTER_IP` is null, then yes, this is correct.

But maybe this should instead be `--host=${KUBE_MASTER_IP:+"https://${KUBE_MASTER_IP}"}` or `${KUBE_MASTER_IP:+"--host=https://${KUBE_MASTER_IP}"}` ?